core-primitives/attestation-handler/src/utils.rs

```diff
@@ -0,0 +1,24 @@
+/*
+	Copyright 2021 Integritee AG and Supercomputing Systems AG
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+
+*/
+
+pub type Hash = sp_core::H256;
+
+pub fn hash_from_slice(hash_slize: &[u8]) -> Hash {
+	let mut g = [0; 32];
+	g.copy_from_slice(hash_slize);
+	Hash::from(&mut g)
+}
```

Comment:
We have a general `utils` crate for functions like this, but unfortunately we moved it into the `pallets` repository. It will be moved back though in the foreseeable future, so this is probably okay for the time being. Once that `utils` crate is back in this repo, we can alos move this `utils.rs` module.

Reply:
Yeah, I'm also not too happy with this. In `core-primitives` there is also already `networking-utils` and `time-utils`. I think it would make sense to move everything into a single `utils` (we can still have submodules there)

Reply:
There is such a general `utils` crate, but it was moved to the pallets repo ([here](https://github.com/integritee-network/pallets/tree/master/primitives/utils/src)) and we use it from there in the worker. So if you want to do it properly with the current state, you'd have to move this function to that `utils` crate. But since we're planning on moving that utils crate back to the worker, it's probably not worth it.

Reply:
Btw, the general `itp-utils` crate is now moved back to the worker repo, so you can move the code there now.

Reply:
I will do that 👍